wiki/History_of_osu!/2015/en.md

```diff
@@ -115,14 +115,14 @@ A few days later (19 November 2015), peppy updated users on the situation in [an
 > - All users which were targeted so far have used the same password for 3+ years.
 > - Many used a very simple password, or shared the password across other services.
 > - All users compromised have had at least one other account compromised in the past.
->
+
```

Comment:
Isn't this going to change the visual style of this?

Reply:
yeah it will be split into three parts, but there's no other way to fix the warning, because for some reason remark doesn't view lists in citation blocks correctly

probably we could return it to the original state and ignore the error, as it's not an issue on our side

Reply:
I think I'd prefer leaving it.